README.rst

```diff
@@ -14,63 +14,82 @@
 
-See this `video for a walk-through. <https://nvidia.github.io/NeMo/>`_
+* **NeMo Core**: fundamental building blocks for all neural models and type system.
+* **NeMo collections**: pre-built neural modules for particular domains such as automatic speech recognition (nemo_asr) and natural language processing (nemo_nlp).
 
 
-**Core Concepts and Features**
+**Introduction**
 
-* `NeuralModule` class - represents and implements a neural module.
-* `NmTensor` - represents activations which flow between neural modules' ports.
-* `NeuralType` - represents types of modules' ports and NmTensors.
-* `NeuralFactory` - to create neural modules and manage training.
-* `Lazy execution` - when describing activation flow between neural modules, nothing happens until an "action" (such as `optimizer.optimize(...)` is called.
-* `Collections` - NeMo comes with collections - related group of modules such as `nemo_asr` (for Speech Recognition) and `nemo_nlp` for NLP
+See `this video <https://nvidia.github.io/NeMo/>`_ for a quick walk-through.
 
 
 **Requirements**
 
 1) Python 3.6 or 3.7
 2) Pytorch 1.2 with GPU support
-3) NVIDIA APEX: https://github.com/NVIDIA/apex
+3) NVIDIA APEX. Install here: https://github.com/NVIDIA/apex
 
 
 **Documentation**
+
 `NeMo documentation <https://nvidia.github.io/NeMo/>`_
 
+See `examples/start_here` to get started with the simplest example. The folder `examples` contains several examples to get you started with various tasks in NLP and ASR.
+
 
 **Getting started**
 
-If desired, you can start with `NGC PyTorch container <https://ngc.nvidia.com/catalog/containers/nvidia:pytorch>`_ which already includes
-requirements above.
+You can use our `NGC PyTorch container <https://ngc.nvidia.com/catalog/containers/nvidia:pytorch>`_ which already includes all the requirements above.
+
+* Pull the docker: ``docker pull nvcr.io/nvidia/pytorch:19.08-py3``
+* Run: ``nvidia-docker run -it --rm -v <nemo_github_folder>:/NeMo --shm-size=1g --ulimit memlock=-1 --ulimit stack=67108864 nvcr.io/nvidia/pytorch:19.08-py3``
+* ``cd /NeMo``
+
+and then continue with the following steps.
+
+**Note**
 
-* You can pull it like so: ``docker pull nvcr.io/nvidia/pytorch:19.08-py3``
-* And then run: ``nvidia-docker run -it --rm -v <nemo_github_folder>:/NeMo --shm-size=1g --ulimit memlock=-1 --ulimit stack=67108864 nvcr.io/nvidia/pytorch:19.08-py3``
-* ``cd /NeMo''
+For step 2 and 3, if you want to use NeMo in development mode, use:
 
-and then continue with the following steps:
+``pip install -e .``
 
+instead of 
 
-1) Clone the repository
-2) Go to nemo folder and then: ``python setup.py install``
-3) Install collections:
-    * ASR collection from `collections/nemo_asr`: 
-        1. ``apt-get install libsndfile1``
-        2. ``python setup.py install``
+``pip install .``
+
+1) Clone the repository ``git clone https://github.com/NVIDIA/NeMo.git``
+2) Go to NeMo folder and install the toolkit:
+
+.. code-block:: bash
+
+	cd NeMo
+	pip install .
```

Comment:
should we change our "reinstall.sh" script too?

Reply:
yes!

Reply:
in a separate PR then